src/components/Text.tsx

```diff
@@ -237,13 +235,11 @@ const linkText = (
     textColor(
       gradientFrom && gradientTo ? 'text-transparent' : color || 'text-accent'
     ),
-    backgroundImage(
-      gradientFrom && gradientTo ? 'bg-gradient-to-r' : undefined
-    ),
-    backgroundClip(gradientFrom && gradientTo ? 'bg-clip-text' : undefined),
-    fontSize('text-sm', 'tiny:text-base'),
+    backgroundImage({ 'bg-gradient-to-r': !!gradientFrom && !!gradientTo }),
```

Comment:
You're using `gradientFrom && gradientTo` combo 3 times here, let's extract it into:
```TS
const withGradients = gradientFrom && gradientTo
```

Reply:
Done